src/pages/view-deck-pages/view-personal-deck-page/view-personal-deck-page.scss

```diff
@@ -10,8 +10,7 @@
     grid-auto-flow: row;
     grid-template-columns: auto min-content;
     grid-template-rows: min-content auto;
-    column-gap: 16px;
-    overflow-wrap: anywhere;
```

Comment:
we probably want to keep this (merge conflict)

Reply:
Oh, good catch; I'll get that updated.